src/components/manager/projects/Project.js

```diff
@@ -106,6 +109,29 @@ class Project extends Component {
                 </>
             )}
 
+            <Typography.Title level={4} style={{marginTop: "1.2em"}}>
```

Comment:
move to tab, using outlined tabs to visually distinguish from other tabs on page & establish visual hierarchy & flow of project management

Reply:
Done